xs2a-gateway-rest-server/src/main/java/de/adorsys/xs2a/gateway/resource/PaymentResource.java

```diff
@@ -18,21 +18,22 @@
 import java.util.UUID;
 
 @RestController
-public class PaymentController extends AbstractController implements PaymentApi {
+public class PaymentResource extends AbstractResource implements PaymentApi {
```

Comment:
I think that `ConsentResource` should be renamed to controller. Not only that's only occurrence of the word, but the term is misleading because Apis also include endpoints for authorization sub-resources.

Reply:
makes sense, renamed